tracee/consts.go

```diff
@@ -31,382 +31,388 @@ const (
-	//	247: "waitid",
-	//	248: "add_key",
-	//	249: "request_key",
-	//	250: "keyctl",
-	//	251: "ioprio_set",
-	//	252: "ioprio_get",
-	//	253: "inotify_init",
-	//	254: "inotify_add_watch",
-	//	255: "inotify_rm_watch",
-	//	256: "migrate_pages",
-	257: "openat",
-	//	258: "mkdirat",
-	259: "mknodat",
-	260: "fchownat",
-	//	261: "futimesat",
-	//	262: "newfstatat",
-	263: "unlinkat",
-	//	264: "renameat",
-	//	265: "linkat",
-	266: "symlinkat",
-	//	267: "readlinkat",
-	268: "fchmodat",
-	269: "faccessat",
-	//	270: "pselect6",
-	//	271: "ppoll",
-	//	272: "unshare",
-	//	273: "set_robust_list",
-	//	274: "get_robust_list",
-	//	275: "splice",
-	//	276: "tee",
-	//	277: "sync_file_range",
-	//	278: "vmsplice",
-	//	279: "move_pages",
-	//	280: "utimensat",
-	//	281: "epoll_pwait",
-	//	282: "signalfd",
-	//	283: "timerfd_create",
-	//	284: "eventfd",
-	//	285: "fallocate",
-	//	286: "timerfd_settime",
-	//	287: "timerfd_gettime",
-	288: "accept4",
-	//	289: "signalfd4",
-	//	290: "eventfd2",
-	//	291: "epoll_create1",
-	292: "dup3",
-	//	293: "pipe2",
-	//	294: "ionotify_init1",
-	//	295: "preadv",
-	//	296: "pwritev",
-	//	297: "rt_tgsigqueueinfo",
-	//	298: "perf_event_open",
-	//	299: "recvmmsg",
-	//	300: "fanotify_init",
-	//	301: "fanotify_mark",
-	//	302: "prlimit64",
-	//	303: "name_tohandle_at",
-	//	304: "open_by_handle_at",
-	//	305: "clock_adjtime",
-	//	306: "sycnfs",
-	//	307: "sendmmsg",
-	//	308: "setns",
-	//	309: "getcpu",
-	310: "process_vm_readv",
-	311: "process_vm_writev",
-	//	312: "kcmp",
-	313: "finit_module",
-	//	314: "sched_setattr",
-	//	315: "sched_getattr",
-	//	316: "renameat2",
-	//	317: "seccomp",
-	//	318: "getrandom",
-	319: "memfd_create",
-	//	320: "kexec_file_load",
-	//	321: "bpf",
-	322: "execveat",
-	//	323: "userfaultfd",
-	//	324: "membarrier",
-	//	325: "mlock2",
-	//	326: "copy_file_range",
-	//	327: "preadv2",
-	//	328: "pwritev2",
-	329: "pkey_mprotect",
-	//	330: "pkey_alloc",
-	//	331: "pkey_free",
-	//	332: "statx",
-	//	333: "io_pgetevents",
-	//	334: "rseq",
-	//	335: reserved
-	//	336: reserved
-	//	337: reserved
-	//	338: reserved
-	//	339: reserved
-	//	340: reserved
-	//	341: reserved
-	//	342: reserved
-	//	343: reserved
-	//	344: reserved
-	//	345: reserved
-	//	346: reserved
-	//	347: reserved
-	//	348: reserved
-	//	349: reserved
-}
+// ProbeType is an enum that describes the mechanism used to attach the event
+type ProbeType uint8
 
-// EventsTracepoints is list of supported tracepoints events, indexed by their ID
-// Currently the item name is the tracepoint category and the actuall tracepoint is always sys_enter
+// Syscall tracepoints are explained here: https://github.com/iovisor/bcc/blob/master/docs/reference_guide.md#8-system-call-tracepoints
+// Kprobes are explained here: https://github.com/iovisor/bcc/blob/master/docs/reference_guide.md#1-kprobes
 // Tracepoints are explained here: https://github.com/iovisor/bcc/blob/master/docs/reference_guide.md#3-tracepoints
```

Comment:
would be better if each comment is adjacent the const it relates to

Reply:
All the events are now grouped togeher, this is why I grouped the comments here

Reply:
yes but you grouped the comments above all the ProbeTypes, instead of every comment above it's related ProbeType